app/main.py

```diff
@@ -1,3 +1,26 @@
+import os
+from typing import Any
+
+
 class CleanUpFile:
-    # write your code here
-    pass
+    def __init__(self, filename: str) -> None:
+        self.filename = filename
+        self.file = None
+
+    def __enter__(self) -> None:
+        self.file = open(self.filename)
+        return self.file
```

Comment:
```suggestion
        return self
```
Twój contextmanager nie ma otwierać pliku, tylko go usuwać po jego zamknięciu